portfolio/src/main/webapp/script.js

```diff
@@ -11,3 +11,24 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+/**
+ * Fetches comments from the servers and adds them to the DOM.
+ */
+async function getComments() {
+  const response = await fetch('/data');
+  const comments = await response.json();
+  const commentsElement = document.getElementById('comments-container');
+  commentsElement.innerHTML = '';
+
+  for (var comment of comments) {
+    commentsElement.appendChild(createListElement(comment))
+  }
+}
+
+/** Creates an <li> element containing text. */
```

Comment:
Tiny nit: it's probably better to have a consistent comment style where you use the blank lines, like:
```
/**
 * Creates an <li> element containing text.
 */
```

Reply:
Got it!